readme.txt

```diff
@@ -0,0 +1,41 @@
+[](https://img.shields.io/badge/Microverse-blueviolet)
+# Erez and Cesar
+> learning to use git and learning good work practises
+![screenshot](./app_screenshot.png)
+Additional description about the project and its features.
+## Built With
+- Major html
+- Frameworks
+- Technologies used
+## vsc text editor
+[Live Demo Link](https://livedemo.com)
+## Getting Started
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
Delete this section and type something if you feel you have some info for the user to get started using this repo!